includes/class-wc-stripe-intent-controller.php

```diff
@@ -810,16 +810,6 @@ public function update_and_confirm_payment_intent( $payment_intent, $payment_inf
 
 		$request = $this->build_base_payment_intent_request_params( $payment_information );
 
-		// Add the updated preferred credit card brand when defined
-		$preferred_brand = $payment_information['payment_method_details']->card->networks->preferred ?? null;
-		if ( isset( $preferred_brand ) ) {
-			$request['payment_method_options'] = [
```

Comment:
When including this parameter, the request fails with a 400 code, and the following information.

```
parameter_unknown - payment_method_options[card][brand]
Received unknown parameter: payment_method_options[card][brand]
```

<details>

```
{
  "capture_method": "automatic",
  "payment_method": "pm_1PM51MDhoeKpknCIpl4x6gwH",
  "mandate_data": {
    "customer_acceptance": {
      "type": "online",
      "online": {
        "user_agent": "WooCommerce Stripe Gateway8.3.0; http://localhost:8082",
        "ip_address": "xxx"
      }
    }
  },
  "payment_method_options": {
    "card": {
      "brand": "cartes_bancaires"
    }
  },
  "shipping": {
    "address": {
      "country": "US",
      "line1": "Street 123",
      "state": "CA",
      "postal_code": "94110",
      "city": "Lecity",
      "line2": ""
    },
    "name": "Francis TheFrench"
  }
}
```

</details>

To replicate it, check out `develop` and follow the steps under "Regression test for reusing a payment intent with co-branded cards" from this PR description.

I'm removing it in this PR to avoid payment failures, and opening an issue to address this separately.

Reply:
GH issue for confirming that we're passing the correct co-branded card information in our requests 

- https://github.com/woocommerce/woocommerce-gateway-stripe/issues/3167

Reply:
Thank you Danae for explaining this and I agree with your decision to remove the code in this PR and address it separately 💯